### DISCUSSION TITLE
Heading position

### DISCUSSION THREAD

**Discussion by vishhvak** [marp-team/marp#327] [Q&A] [answered]
Just started learning about marp. Its great, I already love using it!

That said, how would I go about fixing the title/heading of every slide to the top left, or a fixed position? Right now, the title auto adjusts itself as I keep adding text to a given slide like this - 

![image](https://user-images.githubusercontent.com/30690855/179862506-73bd4946-1838-4856-b6cc-da345de784ff.png)

![image](https://user-images.githubusercontent.com/30690855/179862458-163a444c-96ad-47b0-8c08-a3a2b2c92025.png)

I would like to keep it in a fixed position at the top left for each slide. Any help on how I can achieve this would be really appreciated. Thanks! :)

**Comment by yhatt** [comment, marked as answer]
It's very easy to apply absolute position through inline style:

```markdown
<style>
h1 {
  position: absolute;
  top: 50px;
  left: 50px;
}
</style>

# Lorem ipsum

- Lorem ipsum dolor sit amet consectetur, adipisicing elit. Deleniti similique perspiciatis quis, modi quaerat praesentium animi tempora obcaecati amet possimus, dicta laborum mollitia recusandae! Assumenda voluptas repudiandae accusantium explicabo error.
- Et porro optio laudantium repellendus ipsam natus excepturi hic enim voluptas, veniam animi quisquam ullam, accusantium amet molestias odio aliquam molestiae quia expedita pariatur consequatur nam. Modi ut esse minima!
```

- See also:
  - https://github.com/marp-team/marpit/issues/153
  - https://github.com/marp-team/marp-core/issues/78
  - https://github.com/marp-team/marp-core/issues/177
